pkg/apis/sources/v1/container_conversion_test.go

```diff
@@ -25,10 +25,10 @@ func TestContainerSourceConversionBadType(t *testing.T) {
 	good, bad := &ContainerSource{}, &testObject{}
 
 	if err := good.ConvertTo(context.Background(), bad); err == nil {
-		t.Errorf("ConvertTo() = %#v, wanted error", bad)
+		t.Error("ConvertTo() = nil, wanted error")
```

Comment:
Would it be useful to add the context of the error in the message as well?

Reply:
not my test :) I am fixing nits.